src/CachePlugin.php

```diff
@@ -58,6 +58,19 @@ public function __construct(CacheItemPoolInterface $pool, StreamFactory $streamF
         $this->config = $optionsResolver->resolve($config);
     }
 
+    public static function clientCache(CacheItemPoolInterface $pool, StreamFactory $streamFactory, array $config = [])
+    {
+        // Allow caching of private requests
+        $config['excluded_directives'] = ['no-cache'];
```

Comment:
Or maybe just allow `respect_cache_headers` to be either boolean or array. If boolean true then assume it is all directives. If false then assume it is empty array.